### PR TITLE
Documentation: Updates for "First steps"

### PR DESCRIPTION
**Pull request by adpaco-aws**
### Description of changes: 

Minor updates for the "First steps" section.

### Resolved issues:

Part of #700 

### Testing:

* How is this change tested? N/A

* Is this a refactor change? No.

### Checklist
- [x] Each commit message has a non-empty body, explaining why the change was made
- [x] Methods or procedures are documented
- [x] Regression or unit tests are included, or existing tests cover the modified code
- [x] My PR is restricted to a single feature or bugfix

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: docs/src/tutorial-first-steps.md [hunk 1]
```diff
@@ -63,11 +61,10 @@ To see traces, run:
 
 ```
 kani --visualize src/lib.rs
-open report/html/index.html
 ```
 
-The first command runs Kani and generates the html-based report in `report/`.
-The second command opens that report in your default browser (on mac, on linux desktops try `xdg-open`).
+This command runs Kani and generates the HTML report in `report/html/index.html`.
+Open the report with your favorite browser.
```
Comment (jaisnan) [inline docs/src/tutorial-first-steps.md:67]: "preferred browser" might be a better way to phrase this.

Comment (adpaco-aws) [reply docs/src/tutorial-first-steps.md]: Done.

File: docs/src/tutorial-first-steps.md [hunk 2]
```diff
@@ -171,7 +168,7 @@ If we call this function with too large of a value, it will crash with an assert
 But we know that, that was our intention.
 
 This is the purpose of _proof harnesses_.
-Much like property testing (which would also find this assertion failure as a bug), we need to set up our preconditions, call the function in question, then assert our post conditions.
+Much like property testing (which would also fail in this assertion), we need to set up our preconditions, call the function in question, then assert our postconditions.
```
[the remark this answers is not in the document]
Comment (adpaco-aws) [reply docs/src/tutorial-first-steps.md:171]: I agree with this, and even wanted to go further and have a "Model checking for beginners" appendix in the future. However, there would be a lot to cover there and I think is a bit out of scope for the review.

[the remark this answers is not in the document]
Comment (jaisnan) [reply docs/src/tutorial-first-steps.md:171]: Yes, of course.

**Pull request merged**